scripts/generate_gff3_db.py

```diff
@@ -0,0 +1,127 @@
+"""
```

Comment:
The library portion should end up in a proper package, with generated documentation, etc and not mixed in
with a  program.  The library then be documented and the program serves as an example on writing new programs.
I am not a fan of putting program guts in python packages. 

But not until it's needed.  Please file a ticket to do this, but carry on!